docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md

```diff
@@ -0,0 +1,32 @@
+# Convert PyTorch* QuartzNet to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_QuartzNet}
```

Comment:
I don't see that another document references to this one. So, it seems it's not a part of common documentation.